mule-extensions-api/src/main/java/org/mule/runtime/extension/api/declaration/type/TypeUtils.java

```diff
@@ -40,7 +41,9 @@ private TypeUtils() {}
    */
   public static Collection<Field> getParameterFields(Class<?> declaringType) {
     return getAllFields(declaringType).stream()
-        .filter(field -> (field.getAnnotation(Parameter.class) != null || field.getAnnotation(ParameterGroup.class) != null)
+        .filter(field -> (field.getAnnotation(Parameter.class) != null || field.getAnnotation(ParameterGroup.class) != null
```

Comment:
maybe a method called isParameter(Field field) might be added, and in the filter it could be combined with the method isParameterGroup that you've created, what do you think?

Reply:
agree 👍  adding it.